app/actions/websocket/index.ts

```diff
@@ -104,6 +104,8 @@ async function doReconnect(serverUrl: string, groupLabel?: BaseRequestGroupLabel
 
     dataRetentionCleanup(serverUrl);
 
+    expiredBoRPostCleanup();
```

Comment:
should receive the `serverUrl` as argument.

In addition, I see this function calling remove which calls `dataRetentionCleanPosts`, this same function is also potentially called by the above `dataRetentionCleanup`, just make sure there are no race conditions or anything that could lead to a problem.